lib/coproc.h

```diff
@@ -136,7 +136,15 @@ struct PCI_INFO {
     int device_id;
     int domain_id;
 
-    PCI_INFO(): present(false), bus_id(0), device_id(0),domain_id(0) {}
+    void clear() {
```

Comment:
Who static initialization (as in other `clear()` function done) is not used here?